doc/effects.md

```diff
@@ -79,6 +79,9 @@ functionality inherited by all other effects. This includes:
     removed from the game tree and garbage-collected once the effect completes. Set this to false
     if you plan to reuse the effect after it is finished.
 
+  - Optional user-provided `onComplete` callback, which will be invoked when the effect has just
```

Comment:
It is a bit confusing to have one function that is called `onComplete` and one that is called `onFinish`, not sure how to solve it though.

Reply:
Yeah, I guess that's why Flutter style guide insists on naming convention "onComplete" for callbacks, and "handleComplete" for methods.

Reply:
We could just name it `onFinishCallback`